hoot-services/src/main/java/hoot/services/job/JobType.java

```diff
@@ -43,7 +43,8 @@
     UPLOAD_CHANGESET,
     BULK_ADD,
     BULK_REPLACE,
-    BULK_DIFFERENTIAL;
+    BULK_DIFFERENTIAL,
+    SYNC_WAIT;
```

Comment:
felt a bit odd to add a wait job. what do you think?

Reply:
So on the first run the sync wait command is part of the bulk job, but when a timed out wait is restarted it needs it's own job type because it's the only command running, yes?
If so I feel this makes sense.

Reply:
yup correct